pymeasure/instruments/oxfordinstruments/mercuryitc.py

```diff
@@ -0,0 +1,314 @@
+        get_process=lambda v: v*1e-3
+    )
+
+    control_temperature = Channel.measurement(
+        "READ:DEV:{ch}:TEMP:SIG:TEMP",
+        """Get the control temperature, in Kelvin.""",
+        preprocess_reply=lambda v: v.split(":")[-1].replace("K", "")
+    )
+
+    loop_PID_enabled = Channel.control(
+        "READ:DEV:{ch}:TEMP:LOOP:ENAB",
+        "SET:DEV:{ch}:TEMP:LOOP:ENAB:%s",
+        """Control whether the control-loop heater is controlled by
+        the PID loop (``True``) or the manual heater percentage (``False``).""",
+        check_set_errors=True,
+        preprocess_reply=lambda v: v.split(":")[-1],
+        validator=strict_discrete_set,
+        values={True: "ON", False: "OFF"},
+        map_values=True
+    )
+
+    loop_P = Channel.control(
+        "READ:DEV:{ch}:TEMP:LOOP:P",
+        "SET:DEV:{ch}:TEMP:LOOP:P:%g",
+        """Control the proportional term, P, of the control loop.""",
+        check_set_errors=True,
+        preprocess_reply=lambda v: v.split(":")[-1]
+    )
+
+    loop_I = Channel.control(
+        "READ:DEV:{ch}:TEMP:LOOP:I",
+        "SET:DEV:{ch}:TEMP:LOOP:I:%g",
+        """Control the integral term, I, of the control loop.""",
+        check_set_errors=True,
+        preprocess_reply=lambda v: v.split(":")[-1]
+    )
+
+    loop_D = Channel.control(
+        "READ:DEV:{ch}:TEMP:LOOP:D",
+        "SET:DEV:{ch}:TEMP:LOOP:D:%g",
+        """Control the derivative term, D, of the control loop.""",
+        check_set_errors=True,
+        preprocess_reply=lambda v: v.split(":")[-1]
+    )
+
+    loop_T_setpoint = Channel.control(
+        "READ:DEV:{ch}:TEMP:LOOP:TSET",
+        "SET:DEV:{ch}:TEMP:LOOP:TSET:%g",
+        """Control the control loop temperature setpoint, in Kelvin (in the range
+        0 to 2000).""",
+        check_set_errors=True,
+        preprocess_reply=lambda v: v.split(":")[-1].replace("K", ""),
+        validator=truncated_range,
+        values=[0, 2000]
+    )
+
+    loop_heater_pcnt = Channel.control(
+        "READ:DEV:{ch}:TEMP:LOOP:HSET",
+        "SET:DEV:{ch}:TEMP:LOOP:HSET:%g",
+        """Control the heater percentage when output configured to manual mode
+        (in the range 0 to 100).""",
+        check_set_errors=True,
+        preprocess_reply=lambda v: v.split(":")[-1],
+        validator=truncated_range,
+        values=[0, 100]
+    )
+
+    loop_ramp_rate = Channel.control(
+        "READ:DEV:{ch}:TEMP:LOOP:RSET",
+        "SET:DEV:{ch}:TEMP:LOOP:RSET:%g",
+        """Control the control loop ramp rate, in K/min, if enabled (in the
+        range 0 to 100000).""",
+        check_set_errors=True,
+        preprocess_reply=lambda v: v.split(":")[-1].replace("K/m", ""),
+        validator=truncated_range,
+        values=[0, 100000]
+    )
+
+    loop_ramp_enabled = Channel.control(
+        "READ:DEV:{ch}:TEMP:LOOP:RENA",
+        "SET:DEV:{ch}:TEMP:LOOP:RENA:%s",
+        """Control whether the temperature ramping function is enabled (``True``) or
```

Comment:
As the name is already self evident (therefore we suggest such names), you do not have to spell out, that True enables.
it is sufficient to state, that it is boolean, e. G. With "(bool)" the end.